engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -162,16 +162,22 @@ private void processInput(EntityRef entity, CharacterMovementComponent character
         Quat4f viewRot;
         switch (characterMovementComponent.mode) {
             case WALKING:
-                viewRot = new Quat4f(TeraMath.DEG_TO_RAD * lookYaw, 0, 0);
-                viewRot.rotate(relMove, relMove);
+                if (!playerCamera.isTracked()) {
```

Comment:
Hmmmm. Very unconvinced by this. When cameras will be components it won't be the camera component to see its data change as the player moves his head and walk around the room, but the orientation and position components in the same entity. What should we have then? A tracked flag in both position and orientation components? I would recommend not: if a tracking system is enabled it should take exclusive control of those components and simply write into them. In this context it would be perhaps more appropriate that the LocalPlayerSystem or the CharacterSystem store a tracked flag. That been said: when is the player's head really _not_ tracked? Isn't changing the orientation of a character with the mouse a form of (hand) tracking?